swift_static_framework/swift_static_framework.bzl

```diff
@@ -3,42 +3,13 @@
 load("@build_bazel_apple_support//lib:apple_support.bzl", "apple_support")
 load("@build_bazel_rules_swift//swift:swift.bzl", "SwiftInfo", "swift_library")
 
-_DEFAULT_MINIMUM_OS_VERSION = "10.0"
+_DEFAULT_MINIMUM_OS_VERSION = "11.0"
```

Comment:
Is min iOS 11 required for these changes? If callers want to override to iOS 9 or 10, will this still work?

Reply:
To simplify the number of outputs, because no one is supporting iOS 10 anymore... right?

Reply:
This won't work if callers override it to iOS 10 or below.